cinema_service/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("api/user/", include("user.urls", namespace="user")),
```

Comment:
Ensure that the `cinema.urls` and `user.urls` modules are correctly defined and include the necessary URL patterns for the application. The namespaces should match those defined in the respective `urls.py` files.